src/types/settings.ts

```diff
@@ -18,3 +18,7 @@ export interface UserSettingsResponse {
 export interface AppSettingsQueryResponse {
   appSettings: Array<{ key: string; value: any; public?: boolean }>;
 }
+
+export interface AdminAppSettingsQueryResponse {
+  adminAppSettings: Array<{ key: string; value: any; public?: boolean }>;
```

Comment:
Create an AdminAppSetting with those values and pass it to AdminAppSetting[]
```suggestion

export interface AdminAppSetting {
  key: string;
  value: string | number | boolean | object; 
  public?: boolean;
}

export interface AdminAppSettingsQueryResponse {
  adminAppSettings: AdminAppSetting[];
}
```